src/main/java/uk/ac/ebi/ena/app/MainRunner.java

```diff
@@ -103,7 +103,10 @@ public void run(String... args) {
 
             if (args.length >= 5) {
                 try {
+                    trimInputs(downloadLocation, asperaLocation, accessions, userName);
+                    formatStr = StringUtils.trim(formatStr);
```

Comment:
can reduce a line of code by doing the trim when you pass to enum?
Also, should we not do blank checks on these params? what happens if they are null?

Reply:
StringUtils.trim  handles for NPE so no need for null check.